components/Avatar/avatar.html

```diff
@@ -0,0 +1,189 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar</title>
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="shortcut icon" href="images/favicon-32x32.png" type="image/x-icon">
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <!-- logo -->
+            <a  class="logo" href="/"><div class="nav-logo">
+                LILAC UI
+            </div>
+            </a>
+    
+            <!-- nav links -->
+            <div class="nav-links">
+                <a href="docs.html" class="nav-items"><span>Docs</span></a>
+                <a href="https://github.com/muskan-gupta-0803/LilacUI" class="nav-items"><span>Github</span></a>
+                <a href="#" class="nav-items"><span>
+                    <i class="fas fa-sun"></i>
+                </span></a>
+                <a href="#" class="nav-items"><span>Item 4</span></a>
+            </div>
+        </nav>
+    </header>
+
+    <!-- main container -->
+
+    <div class="doc-container">
+    <!-- every page same menu -->
+        <aside class="menu-all">
+            <ul class="doc-list ma-list">
+                <h5 class="ma-heading">Get Started</h5>
+                <li class="ma-item"><a class="ma-link" href="docs.html">Introduction </a></li>
+                <li class="ma-item"><a class="ma-link" href="">Utilities </a></li>
+                <h5 class="ma-heading">Components</h5>
+                <li class="ma-item"><a class="ma-link" href="">List item </a></li>
+                <li class="ma-item"><a class="ma-link" href="avatar.html">Avatar </a></li>
+                <li class="ma-item"><a class="ma-link" href="">List item </a></li>
+                <li class="ma-item"><a class="ma-link" href="">List item </a></li>
+                <li class="ma-item"><a class="ma-link" href="">List item </a></li>
+                <li class="ma-item"><a class="ma-link" href="">List item </a></li>
+            </ul>
+        </aside>
+
+
+    <!-- main body area -->
+        <main class="main">
+            <section class="intro">
+                <h1 class="para-heading">Avatar</h1>
+                <p class="doc-para">
+                    The avatar component is mainly used to display user profile pictures.
+                </p>
+
+                <!-- STANDARD AVATAR -->
+                <h2 class="para-heading" id="std-avatar">Standard Avatar</h2>
+                <p class="doc-para">
+                    A standard avatar component is typically circular in shape.
+                    You can use the following code to create this avatar.
+                </p>
+                <div class="comp-container">
+                    <!-- Basic avatars -->
+                    <img class="avatar img-circle " src="https://i.pravatar.cc/150?img=1" alt="" /> 
```

Comment:
`img` should have description `alt` text unless purely used for decoration purpose.

Reply:
done. Thanks for such a detailed review.